src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -193,6 +193,39 @@ public void should_attach_exception_record_to_case_by_attach_to_case_reference()
         verifyExceptionRecordAttachesToCase(null);
     }
 
+    @Test
+    public void should_attach_exception_record_to_case_by_searchCaseReference_and_attachToCaseReference()
```

Comment:
how are both `searchCaseReference` and `attachToCaseReference` going to be set ? Is it by CCD but I am not sure how both fields will be populated

Reply:
yeah, it's not realistic. I'll update the test to send `searchCaseReference` value, `searchCaseReferenceType` as null and without `attachToCaseReference` field. Is that ok?